packages/palette-charts/src/elements/BarChart/BarChart.test.tsx

```diff
@@ -35,6 +38,14 @@ const mockBars = [
 ]
 
 describe("BarChart", () => {
+  const mockUseIntersectionObserver = useIntersectionObserver as jest.Mock
+
+  beforeAll(() => {
+    mockUseIntersectionObserver.mockImplementation(() => ({
+      ref: createRef(),
+    }))
+  })
+
```

Comment:
if you wanna walk the extra mile to have a proper test for it later, [example](https://github.com/artsy/volt-v2/blob/main/src/pages/conversations/%5BconversationId%5D/Components/__tests__/ConversationMessages.test.tsx#L181)